<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Standardna kola</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Button_LINE">
         <property name="text">
          <string>LINE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_NOT">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/NOT.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_AND">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/AND.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_OR">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/OR.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_XOR">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/XOR.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_NAND">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/NAND.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_NOR">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/NOR.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_XNOR">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/ikone_dugmici/D:/IKONE_KOLA/XNOR.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonPrekidac">
         <property name="text">
          <string>Prekidac</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_RUKA">
         <property name="text">
          <string>RUKA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pomerajVise">
         <property name="text">
          <string>MOVEMORE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear_All"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuSracunaj">
    <property name="title">
     <string>Sracunaj</string>
    </property>
    <addaction name="actionCelu_tabelicu"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionSave_User_Defined"/>
    <addaction name="actionLoad_UDS"/>
    <addaction name="actionOtvori_UDS"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSracunaj"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClear_All">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCelu_tabelicu">
   <property name="text">
    <string>Celu tabelicu</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionSave_User_Defined">
   <property name="text">
    <string>Save User Defined</string>
   </property>
  </action>
  <action name="actionLoad_UDS">
   <property name="text">
    <string>Load UDS</string>
   </property>
  </action>
  <action name="actionOtvori_UDS">
   <property name="text">
    <string>Otvori UDS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
